### PR TITLE
OCM-6140 | feat: allow customization of the domain prefix when creating a cluster and prints it out in the describe cluster command

### PR DESCRIPTION
**Pull request by machi1990**

### CODE REVIEW

File: cmd/ocm/create/cluster/cmd.go [hunk 1]
```diff
@@ -174,6 +176,18 @@ func init() {
 	arguments.SetQuestion(fs, "version", "OpenShift version:")
 	Cmd.RegisterFlagCompletionFunc("version", arguments.MakeCompleteFunc(getVersionOptions))
 
+	fs.StringVar(
+		&args.domainPrefix,
+		"domain-prefix",
+		"",
+		"An optional unique domain prefix of the cluster. This will be used when generating a "+
+			"sub-domain for your cluster on openshiftapps.com. It must be unique and consist of "+
```
Comment (tzvatot) [inline cmd/ocm/create/cluster/cmd.go:184]: If we ever change the URL from openshiftapps.com to something else, it means every customer that is using an older version of the CLI would see incorrect help message. That said, I don't have a workaround/alternative other than maybe writing something vague w/o specifying the exact domain. 
Anyhow, not a strong objection from writing it, just raising the awareness.

Comment (machi1990) [reply cmd/ocm/create/cluster/cmd.go:184]: That's a good point; I basically went with the help message that was in the `.name`.

Comment (machi1990) [reply cmd/ocm/create/cluster/cmd.go:184]: I could drop the `openshiftapps.com` from the text all together.

Comment (machi1990) [reply cmd/ocm/create/cluster/cmd.go]: done

**Comment by machi1990** [comment]
@tzvatot Please give this another round of reviews. Thanks

File: cmd/ocm/create/cluster/cmd.go [hunk 2]
```diff
@@ -174,6 +176,18 @@ func init() {
 	arguments.SetQuestion(fs, "version", "OpenShift version:")
 	Cmd.RegisterFlagCompletionFunc("version", arguments.MakeCompleteFunc(getVersionOptions))
 
+	fs.StringVar(
+		&args.domainPrefix,
+		"domain-prefix",
+		"",
+		"An optional unique domain prefix of the cluster. This will be used when generating a "+
```
Comment (tzvatot) [inline cmd/ocm/create/cluster/cmd.go:183]: Would be nice to know what will be the behaviour  (default) if user does not provide it. WDYT?

Comment (machi1990) [reply cmd/ocm/create/cluster/cmd.go:183]: Sounds good.

Comment (machi1990) [reply cmd/ocm/create/cluster/cmd.go]: done

**Review by tzvatot — APPROVED** [review]
LGTM

**Pull request merged**